public/app/features/dashboard/components/SaveDashboard/forms/SaveDashboardForm.tsx

```diff
@@ -30,32 +31,32 @@ export const SaveDashboardForm: React.FC<SaveDashboardFormProps> = ({ dashboard,
     >
       {({ register, errors }) => (
         <>
-          <Forms.Field label="Changes description">
-            <Forms.TextArea
-              name="message"
-              ref={register}
-              placeholder="Add a note to describe your changes..."
-              autoFocus
-            />
-          </Forms.Field>
-          {hasTimeChanged && (
-            <Forms.Field label="Save current time range" description="Dashboard time range has changed">
-              <Forms.Switch
+          <div className="gf-form-group">
```

Comment:
Maybe let's not use "legacy" class names" and `VerticalGroup` with sizing instead?

Reply:
Yea I know, but we are missing a component for this. We need a FieldGroup component with consistent margin, not sure VerticalGroup with manual spacing specified in every form is a good solution if we want consistent form spacings. 

https://github.com/grafana/grafana-enterprise/pull/445 

But wanted to keep this PR small, in order to cherry pick this into 6.7 so tried to use the existing form group class